services/blockchain-app-registry/shared/database/mysqlKVStore.js

```diff
@@ -29,7 +29,7 @@ const getKeyValueStoreIndex = () => getTableInstance(
 	MYSQL_ENDPOINT,
 );
 
-const set = async (key, value) => {
+const set = async (key, value, dbTrx) => {
```

Comment:
All changes copied from indexer: https://github.com/LiskHQ/lisk-service/blob/1467-add-tests-for-blockchain-app-registry-microservice/services/blockchain-indexer/shared/database/mysqlKVStore.js